ui/images/pods-form-placeholder.svg

```diff
@@ -0,0 +1,46 @@
+<svg width="530" height="253" viewBox="0 0 530 253" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@nicdford Can we get a faux submit button on the right side of the form image here?

Reply:
Absolutely!